src/WebJobs.Script/Config/FunctionsHostingConfigOptions.cs

```diff
@@ -183,6 +183,14 @@ internal bool ThrowOnMissingFunctionsWorkerRuntime
             }
         }
 
+        internal bool ThrowOnFunctionsWorkerRuntimeMismatchWithMetadataFromPayload
+        {
+            get
+            {
+                return GetFeature(RpcWorkerConstants.ThrowOnFunctionsWorkerRuntimeMismatchWithMetadataFromPayload) == "1";
```

Comment:
Perhaps use the GetFeatureAsBooleanOrDefault helper which seems to be what many of the other boolean configs are using?

Reply:
Also, perhaps we name this config WorkerRuntimeStrictValidationEnabled

Reply:
Renamed to `WorkerRuntimeStrictValidationEnabled` as suggested.